packages/dialtone-css/postcss/dialtone-docs.cjs

```diff
@@ -49,6 +49,76 @@ function generateExclusionRegex () {
 const documentation = {};
 const CSSVarRegex = /var\(([^),]+)\)/g;
 
+/**
+ * Metadata rules for utility classes
+ * Maps patterns to metadata that should be added to the documentation
+ * Based on eslint-plugin-dialtone rules
+ */
+const metadataRules = [
+  // Typography utilities (discouraged per recommend-typography-style.js eslint rule)
+  {
+    pattern: /^d-(fw-|ff-|fs-|lh-)/,
+    metadata: {
+      deprecated: false,
+      discouraged: true,
+      category: 'typography',
+      reason: 'Individual typography utilities are discouraged in favor of composed typography classes',
+      alternatives: ['d-headline-*', 'd-body-*', 'd-label-*', 'd-code-*'],
+      docs: 'https://dialtone.dialpad.com/design/typography/#api',
+    },
+  },
+  // Base colors (deprecated per deprecated-base-color-classes.js eslint rule)
+  {
+    pattern: /^d-(bgc|fc|bc|divide)-\w+-\d{2,4}$/,
+    metadata: {
+      deprecated: true,
+      discouraged: true,
+      category: 'color',
+      reason: 'Base color utilities are deprecated and will be removed in the future',
+      alternatives: [],
+      docs: 'https://dialtone.dialpad.com/utilities/backgrounds/color.html',
```

Comment:
Currently specific to `backgrounds`. Can it include pointing to the other CSS Utility doc pages for foreground color, border color, divide color?